File: wlada-si.21.19.getout-e707bbd07835/Aplikacija/SWEproj/backend/GETOut/Models/Galerija.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

#nullable disable

namespace GETOut.Models
{
    [Table("galerija")]
    public partial class Galerija
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("kafic_id")]
        public int KaficId { get; set; }

        [Column("url")]
        public string Url { get; set; }

    }
}

File: wlada-si.21.19.getout-e707bbd07835/Aplikacija/SWEproj/frontend/src/app/models/galerija.ts
export class galerija
{
    id: number;
    url: string;
    kaficId: number;

    constructor(id: number, url: string, kaficId: number)
    {
        this.id = id;
        this.url = url;
        this.kaficId = kaficId;
    }
}